spec-dtslint/observables/combineLatest-spec.ts

```diff
@@ -37,31 +37,31 @@ it('should accept union types', () => {
 });
 
 it('should accept 1 param and a result selector', () => {
-  const o = combineLatest(a$, () => new A()); // $ExpectType Observable<A>
+  const o = combineLatest(a$, (a) => new A()); // $ExpectType Observable<A>
```

Comment:
nitpick: Specifying the parameter here does nothing, ATM, but we could improve the tests by using them:

```ts
const o = combineLatest(a$, (a) => [a] as const); // $ExpectType Observable<[A]>
```